src/unstract/sdk/audit.py

```diff
@@ -33,23 +24,26 @@ def push_usage_data(
         self,
         platform_api_key: str,
         token_counter: TokenCountingHandler = None,
-        workflow_id: str = "",
-        execution_id: str = "",
-        external_service: str = "",
+        model_name: str = "",
         event_type: CBEventType = None,
+        **kwargs,
     ) -> None:
         """Pushes the usage data to the platform service.
 
         Args:
+            platform_api_key (str): The platform API key.
             token_counter (TokenCountingHandler, optional): The token counter
-              object. Defaults to None.
+                object. Defaults to None.
+            model_name (str, optional): The name of the model.
+                Defaults to "".
+            event_type (CBEventType, optional): The type of the event. Defaults
+                to None.
             workflow_id (str, optional): The ID of the workflow. Defaults to "".
```

Comment:
Do you want to list these args here? Ideally the args is just kwargs. You can list the expected kwargs. Explicitly mentioning these as args gives a picture as it they are unpacked here, which is not the case.

Reply:
@gaya3-zipstack I added it so that we know that these args are sent just by looking into the doc string.

Reply:
I agree. All I am saying is that we could do something like
Args:
            kwargs:
                Expected kwargs:
                    - adapter_instance_id
                    - workflow_id
                    - ....

Reply:
Done